### PR TITLE
update volcano.sh/volcano to volcano.sh/apis

### PR DESCRIPTION
**Pull request by Thor-wl**
Signed-off-by: Thor-wl <1187526662@qq.com>

### CODE REVIEW

**Review by terrytangyuan — APPROVED** [review]
PTAL 

/lgtm

File: go.mod [hunk 1]
```diff
@@ -17,7 +17,7 @@ require (
 	k8s.io/client-go v0.16.9
 	k8s.io/code-generator v0.16.9
 	k8s.io/kube-openapi v0.0.0-20190816220812-743ec37842bf
-	volcano.sh/volcano v0.4.0
+	volcano.sh/apis v1.16.15
```
Comment (shinytang6) [inline go.mod:20]: l think we also need to update the gomod dependency?

File: pkg/controller.v1/common/job_controller.go [hunk 2]
```diff
@@ -26,8 +26,8 @@ import (
 	"k8s.io/client-go/tools/cache"
 	"k8s.io/client-go/tools/record"
 	"k8s.io/client-go/util/workqueue"
-	"volcano.sh/volcano/pkg/apis/scheduling/v1beta1"
-	volcanoclient "volcano.sh/volcano/pkg/client/clientset/versioned"
+	"volcano.sh/apis/pkg/apis/scheduling/v1beta1"
+	volcanoclient "volcano.sh/apis/pkg/client/clientset/versioned"
```
Comment (shinytang6) [inline pkg/controller.v1/common/job_controller.go:30]: There is another place using volcano.sh/volcano directly.

link: [https://github.com/kubeflow/common/blob/master/pkg/controller.v1/common/job.go#L18](https://github.com/kubeflow/common/blob/master/pkg/controller.v1/common/job.go#L18)

File: go.mod (hunk 1 above)
Comment (Thor-wl) [reply go.mod:20]: Yes, so i close the pr and submit a new one

Comment (terrytangyuan) [reply go.mod:20]: Why not updating this PR directly?

Comment (Thor-wl) [reply go.mod:20]: Is there any buttons or labels available to restart the CI test?

Comment (terrytangyuan) [reply go.mod:20]: When you push a new commit it will trigger new builds.

Comment (Thor-wl) [reply go.mod:20]: OK, Thanks.

**Pull request closed without merging**